chart/values.yaml

```diff
@@ -289,6 +289,7 @@ workers:
 
 # Airflow scheduler settings
 scheduler:
+  replicas: 1
```

Comment:
Could you add a comment here about Mysql5 or Airflow 1.10

Reply:
Sure thing

Reply:
@ashb fixed